cmd/nvidia-dra-plugin/device_state.go

```diff
@@ -38,17 +38,19 @@ type OpaqueDeviceConfig struct {
 }
 
 type DeviceConfigState struct {
-	MpsControlDaemonID string `json:"mpsControlDaemonID"`
+	MpsControlDaemonID string               `json:"mpsControlDaemonID"`
+	GpuConfig          *configapi.GpuConfig `json:"deviceConfig,omitempty"`
```

Comment:
why not name this `DeviceConfig` if we're exposing it as `deviceConfig` in the json representation?

Reply:
whoops! Thanks for catching that. it should be `gpuConfig`